js/package.json

```diff
@@ -34,24 +34,28 @@
     "@babel/node": "^7.10.5",
     "@babel/plugin-transform-runtime": "^7.10.5",
     "@babel/preset-env": "^7.10.4",
-    "@ledgerhq/hw-transport-node-hid": "^5.19.1",
+    "@ledgerhq/hw-transport-node-hid": "^6.3.0",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^26.1.0",
     "bip32": "^2.0.5",
     "bip39": "^3.0.2",
+    "chai": "^4.3.4",
     "core-js": "^3.6.5",
     "crypto-js": "4.0.0",
+    "elliptic": "^6.5.4",
     "eslint": "^7.4.0",
     "eslint-config-airbnb-base": "^14.2.0",
     "eslint-plugin-import": "^2.22.0",
     "index.js": "^0.0.3",
-    "jest": "^26.1.0",
+    "jest": "^27.0.6",
     "jest-serial-runner": "^1.1.0",
+    "jssha": "^3.2.0",
     "prettier": "^2.0.5",
     "secp256k1": "^4.0.1"
   },
   "scripts": {
-    "build": "babel src  --copy-files --out-dir dist"
+    "build": "babel src  --copy-files --out-dir dist",
```

Comment:
This line causes `test_zemu` [1] to fail in CI.

[1] https://app.circleci.com/pipelines/github/onflow/ledger-app-flow/207/workflows/a6fea794-a991-4417-9857-336739e80622/jobs/643?invite=true#step-106-67

Reply:
I know about this. The core issue is that we need to resolve `@onflow/ledger` (required by Zemu tests) as `/js` folder. We are working on this. But you can safely review the rest of the code.